cmsimple/tplfuncs.php

```diff
@@ -249,14 +249,28 @@ function searchbox()
 }
 
 
+/**
+ * Returns a link to the homepage.
+ *
+ * @param string $img Optional image file name, searched in template folder.
+ *
+ * @return string HTML
+ */
+function homelink($img = '') 
+{
+    //TODO why not use ml() ?
```

Comment:
Yes, would make sense to use `ml()` here, and to add a respective entry to the language files.